scanners/cordova/cordova.go

```diff
@@ -283,7 +283,7 @@ func (*Scanner) DefaultOptions() models.OptionNode {
 
 func (scanner *Scanner) Configs(_ bool) (models.BitriseConfigMap, error) {
 	configBuilder := models.NewDefaultConfigBuilder()
-	configBuilder.AppendStepListItemsTo(models.PrimaryWorkflowID, steps.DefaultPrepareStepList(false)...)
+	configBuilder.AppendStepListItemsTo(models.PrimaryWorkflowID, steps.DefaultPrepareStepList(steps.PrepareListParams{})...)
```

Comment:
Can you please add the isPrivateRepository parameter (see the android one for reference)?